pybossa/jobs.py

```diff
@@ -1358,6 +1359,22 @@ def check_and_send_task_notifications(project_id, conn=None):
         info = dict(project_name=project.name,
                     n_available_tasks=n_remaining_tasks)
         notify_task_progress(info, email_addr)
+
+        if webhook:
+            current_app.logger.info(u'Project {} the number of incomplete tasks: {}, \
+                                drops equal to or below target remaining: {}, hitting webhook url: {}'
+                                .format(project_id, n_remaining_tasks, target_remaining, webhook))
+            try:
+                headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
+                data = dict(project_id=project_id,
+                            project_name=project.name,
+                            remianing_tasks=n_remaining_tasks,
+                            target_remaining=target_remaining)
+                webhook_response = requests.post(webhook, data=json.dumps(data), headers=headers)
+            except Exception:
+                current_app.logger.warning(u'An error occured while posting to project {} webhook {}'
+                                           .format(project_id, webhook))
```

Comment:
if there's exception on webhook post, value of webhook can be reset to null/empty(and owner may be notified on this) so as to avoid server running into exception each time reminder check passed.

Reply:
agree, will send email to owner to notify webhook failure and reset it to empty